src/overlays/actors/ovl_En_Ani/z_en_ani.c

```diff
@@ -21,39 +45,311 @@ const ActorInit En_Ani_InitVars = {
     (ActorFunc)EnAni_Update,
     (ActorFunc)EnAni_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ani/func_809B0370.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ani/EnAni_Init.s")
+static ColliderCylinderInit cylinderInitData = {
+    0x0A, 0x00, 0x11, 0x39,
+    0x10, 0x01, 0x00, 0x00, 
+    0x00, 0x00, 0x00, 0x00,   
+    0x00000000, 
+    0x00, 0x00, 0x00, 0x00, 
+    0xFFCFFFFF, 
+    0x00, 0x00, 0x00, 0x00, 
+    0x00, 0x01, 0x01, 0x00, 
+    0x001E, 0x0028, 0x0000,
+};
+
+static InitChainEntry initChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 10, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_F4, 850, ICHAIN_STOP),
+};
+
+static Vec3f EnAniVec = {
+    800.0f,
+    500.0f,
+    0.0f,
```

Comment:
I think Vec3f initializations like that look much better on a single line, so I suggest removing the trailing comma and running clang-format on this file again.
```suggestion
    0.0f
```